app/src/main/java/com/jerboa/Utils.kt

```diff
@@ -1568,3 +1633,9 @@ fun canMod(
         false
     }
 }
+
+fun futureDaysToUnixTime(days: Int?): Int? {
```

Comment:
Copied from lemmy-ui. The back end requires a unix epoch, but the front end should just use days in its fields.